.gitignore

```diff
@@ -5,12 +5,12 @@
 save*.yaml
 *~
 *#
-/test-resources/RobotBuilderTestProject/*
+/test-resources/
```

Comment:
We probably do not need this in the gitignore anymore.

Reply:
not necessary for new clones, but people would still likely have it in their working tree, so I think it's better to ignore it.